tracing-attributes/tests/async_fn.rs

```diff
@@ -66,6 +66,101 @@ fn async_fn_nested() {
     handle.assert_finished();
 }
 
+#[test]
+fn async_fn_bindings() {
+    #[instrument]
+    async fn test_async_fn_bindings((a, b): (u64, u64)) {
+        let _ = a;
+        let _ = b;
+    }
+
+    let span = span::mock().named("test_async_fn_bindings");
+    let (subscriber, handle) = subscriber::mock()
```

Comment:
looks like this needs to be rebased to track the trait renaming that just landed.